src/pages/en/core-concepts/astro-components.md

```diff
@@ -194,18 +194,18 @@ An Astro component can define and accept props. These props then become availabl
 
 Here is an example of a component that receives a `greeting` prop and a `name` prop. Notice that the props to be received are destructured from the global `Astro.props` object.
 
-```astro
+```astro /(?<!//.*)({?(greeting|name)}?)/ "Astro.props"
```

Comment:
I think this might be a bit overkill

Reply:
Thank you for the feedback! What part do you consider to be an overkill and which alternative would you prefer here?

Reply:
Ok, I experimented with it a bit and think I've found a better version: I removed the variable name marking and just left the main source of all data (`Astro.props`) marked, because this is what's being introduced in that code snippet.

What do you think @kevinzunigacuellar?

Reply:
I was thinking about removing all the highlights but I like the new version better! 

Good to merge!